OpenRA.Game/Settings.cs

```diff
@@ -336,9 +336,16 @@ public class ChatSettings
 		public bool ConnectAutomatically = false;
 	}
 
+	public class LobbySettings
+	{
+		public string Faction = "Random";
```

Comment:
Just so it won't be forgotten, once we have all this merged, #7213 should be within grasp, which would be a significant first step for campaign support, in my opinion.